### PR TITLE
fix ci/e2e code to get local ocm provider

### PR DESCRIPTION
**Pull request by AsherShoshan**

### CODE REVIEW

File: ci/e2e/account_roles_files/main.tf [hunk 1]
```diff
@@ -1,23 +1,35 @@
+#
```
Comment (osherdp) [inline ci/e2e/account_roles_files/main.tf:1]: delete

File: ci/e2e/terraform_provider_ocm_files/main.tf [hunk 2]
```diff
@@ -1,16 +1,18 @@
+#
```
Comment (osherdp) [inline ci/e2e/terraform_provider_ocm_files/main.tf:1]: delete

File: ci/e2e/terraform_provider_ocm_files/variables.tf [hunk 3]
```diff
@@ -19,3 +19,18 @@ variable account_role_prefix {
...
 }
+
+variable aws_region {
+    type = string
+    default = "us-east-1"
+}
+
+variable aws_availability_zones {
+    type = string
+    default = "us-east-1a"
```
Comment (osherdp) [inline ci/e2e/terraform_provider_ocm_files/variables.tf:30]:
```suggestion
    type      = list(string)
    default = ["us-east-1a"]
```

Comment (osherdp) [inline ci/e2e/terraform_provider_ocm_files/variables.tf:30]:
```suggestion
    type    = list(string)
    default = ["us-east-1a"]
```

File: ci/e2e/terraform_provider_ocm_files/main.tf [hunk 4]
```diff
@@ -1,16 +1,18 @@
...
     ocm = {
-      version = "0.0.2"
+      version = ">= 0.0.1"
       source  = "terraform.local/local/ocm"
     }
   }
 }
-
+      # version = " 0.0.3"
+      # source  = "terraform-redhat/ocm"
```
Comment (osherdp) [inline ci/e2e/terraform_provider_ocm_files/main.tf:15]: delete commented-out code

File: ci/e2e/account_roles_files/variables.tf [hunk 5]
```diff
@@ -1,15 +1,21 @@
...
-    default = "staging"
+    default = "4.13"
 }
 
+variable account_role_prefix {
+    type = string
+}
 
-variable rosa_openshift_version {
+variable token {
```
Comment (osherdp) [inline ci/e2e/account_roles_files/variables.tf:15]: for the sake of clarity, let's call it:
```suggestion
variable ocm_token {
```

same goes for ``url``. providing those variables for a user of those files makes it not clear enough what token or url to provide without reading the code

Comment (osherdp) [inline ci/e2e/account_roles_files/variables.tf:15]: for the sake of clarity, let's call it:
```suggestion
variable ocm_token {
```

same goes for ``url``. providing those variables for a user of those files makes it not clear enough what token or url to provide without reading the code

File: ci/e2e/account_roles_files/main.tf [hunk 6]
```diff
@@ -1,23 +1,35 @@
...
   create_account_roles = true
 
-  account_role_prefix = var.account_role_prefix
-  ocm_environment = var.ocm_env
-  rosa_openshift_version=var.rosa_openshift_version
+  account_role_prefix =  var.account_role_prefix
+  ocm_environment =  var.ocm_environment
+  rosa_openshift_version=  var.openshift_version
+  account_role_policies=data.ocm_policies.all_policies.account_role_policies
+  operator_role_policies=data.ocm_policies.all_policies.operator_role_policies
```
Comment (osherdp) [inline ci/e2e/account_roles_files/main.tf:34]: nit:
```suggestion
  account_role_prefix    = var.account_role_prefix
  ocm_environment        = var.ocm_environment
  rosa_openshift_version = var.openshift_version
  account_role_policies  = data.ocm_policies.all_policies.account_role_policies
  operator_role_policies = data.ocm_policies.all_policies.operator_role_policies
```

**Review by osherdp — CHANGES_REQUESTED** [review]

**Comment by osherdp** [comment]
@AsherShoshan please give a more descriptive title and description for this PR. same goes for the commit message

File: ci/e2e/account_roles_files/variables.tf (hunk 5 above)
Comment (AsherShoshan) [reply ci/e2e/account_roles_files/variables.tf]: then u'll have to change all main.tf code as well...

Comment (osherdp) [reply ci/e2e/account_roles_files/variables.tf]: just in one place:
```
provider "ocm" {
  token = var.ocm_token
  url   = var.ocm_url
}
```

**Review by osherdp — APPROVED** [review]

**Pull request merged**